api/test/robot_tests/front_ends/front_ends.robot

```diff
@@ -13,86 +13,13 @@ Suite Teardown      Close All Browsers
+    Wait Until Page Contains    Decentralization made easy    10
 
-GET /home/index.html
-    Open Massa Station Page    ${API_URL}/home/index.html
-    Page Should Contain    Which Plugins
-    Page Should Contain    Registry
-    Page Should Contain    Web On Chain
-    Page Should Contain    Manage plugin
-
-GET /store/
-    Open Massa Station Page    ${API_URL}/store/
-    Page Should Contain    Plugin Manager
-    Page Should Contain    Install a plugin
-    Page Should Contain    Install a plugin using .zip URL
-
-GET /store/index.html
-    Open Massa Station Page    ${API_URL}/store/index.html
-    Page Should Contain    Plugin Manager
-    Page Should Contain    Install a plugin
-    Page Should Contain    Install a plugin using .zip URL
-
-GET /search/
-    Open Massa Station Page    ${API_URL}/search/
-    Page Should Contain    Registry
-    Page Should Contain    Browse decentralized websites
-    Page Should Contain    Website name
-    Page Should Contain    Address
-    Page Should Contain    URL
-
-GET /search/index.html
-    Open Massa Station Page    ${API_URL}/search/index.html
-    Page Should Contain    Registry
-    Page Should Contain    Browse decentralized websites
-    Page Should Contain    Website name
-    Page Should Contain    Address
-    Page Should Contain    URL
-
-GET /websiteUploader/
-    Open Massa Station Page    ${API_URL}/websiteUploader/
-    Page Should Contain    Decentralized website storage
-    Page Should Contain    Upload a website
-    Page Should Contain    On wallet
-    Page Should Contain    Website Name
-    Page Should Contain    Use alphanumerical characters and lowercase
-    Page Should Contain    Website File
-    Page Should Contain Button    website-upload
-    Page Should Contain Button    file-select-button
-
-GET /websiteUploader/index.html
-    Open Massa Station Page    ${API_URL}/websiteUploader/index.html
-    Page Should Contain    Decentralized website storage
-    Page Should Contain    Upload a website
-    Page Should Contain    On wallet
-    Page Should Contain    Website Name
-    Page Should Contain    Use alphanumerical characters and lowercase
-    Page Should Contain    Website File
-    Page Should Contain Button    website-upload
-    Page Should Contain Button    file-select-button
-
-# Error cases
-
-GET /home/{resource} with invalid resource
-    ${response}=    GET    ${API_URL}/home/invalid    expected_status=${STATUS_NOT_FOUND}
-
-GET /store/{resource} with invalid resource
-    ${response}=    GET    ${API_URL}/store/invalid    expected_status=${STATUS_NOT_FOUND}
-
-GET /search/{resource} with invalid resource
-    ${response}=    GET    ${API_URL}/search/invalid    expected_status=${STATUS_NOT_FOUND}
-
-GET /websiteUploader/{resource} with invalid resource
-    ${response}=    GET    ${API_URL}/websiteUploader/invalid    expected_status=${STATUS_NOT_FOUND}
+GET /web/index
+    Open Massa Station Page    ${API_URL}/web/index
```

Comment:
Shouldn't that be :
```suggestion
    Open Massa Station Page    ${API_URL}/web/index.html
```

Reply:
no, react router take the lead on the path